homeassistant/components/blink/camera.py

```diff
@@ -85,6 +92,12 @@ def motion_detection_enabled(self) -> bool:
         """Return the state of the camera."""
         return self._camera.arm
 
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Update camera data."""
+        self._attr_available = True
```

Comment:
So this is always available? I think you should overwrite the avaialble property to True

Reply:
If we get a timeout on any of the commands above, it is set to false 
https://github.com/mkmer/HomeAssistant_Core/blob/2be65a64347221381634b9f05ba839949e293c16/homeassistant/components/blink/camera.py#L79.  
This is the only "indication" that the data is back in sync and allow new commands to be sent (at least that's how I though it worked)

Reply:
Well, I think you need to rethink that part since available is overwritten:
https://github.com/home-assistant/core/blob/af0b53cc79e519df190beb41003e7218fda19c00/homeassistant/helpers/update_coordinator.py#L493-L496